samples/Tenant/GetTenantViews.ps1

```diff
@@ -0,0 +1,64 @@
+[CmdletBinding()]
```

Comment:
The sample should use our commandlet as an example rather than showing how to invoke-webrequest with tenant id ?

Reply:
Done, updated the sample use cmdlets to operate the workflows.